tests/admin/users/updateUser.test.ts

```diff
@@ -38,28 +38,38 @@ describe('PUT /admin/users/:userId', () => {
     await database.user.deleteMany();
   });
 
+  it('should work if body is incomplete', () =>
+    request(app)
+      .patch(`/admin/users/${user.id}`)
+      .send({
+        type: 'player',
+        permissions: [],
+      })
+      .set('Authorization', `Bearer ${adminToken}`)
+      .expect(200));
+
```

Comment:
ça serait bien de rajouter un test pour vérifier que la place n'a pas été reset. Pour se faire, je déplacerais ce test ligne 104 (après le `it should update the user`) car l'user aura une place attitrée. Ainsi, on pourra faire un check `expect(body.place).to.be.equal(validBody.place);`